### ISSUE TITLE
[et] The tool doesn't handle non-terminal type stdout and stderr correctly.

### ISSUE THREAD

**Issue by chinmaygarde** [flutter/flutter#147936]
The tool want to display a progress bar and the latest target being compiled prettily to the terminal. And it does that. However for non terminal type stdout/stderr (IIRC checked via `isatty` and `GetConsoleScreenBufferInfo`), the output is a bit of a mess. For instance, via Gnu parallel, invoking `parallel et build -c ::: host_profile_arm64 android_profile_arm64` will dump a whole mess of text that is hard to parse. The same happens when the build system is invoked via editor build system integration.

An instance of the text output:
<img width="1236" alt="Screenshot 2024-05-07 at 11 22 17 AM" src="https://github.com/flutter/flutter/assets/44085/484b7f74-8afc-4c24-90a7-0bfb29181828">

**Comment by chinmaygarde** [comment]
I don't know if https://github.com/flutter/flutter/issues/147903 is tracking a flag that explicitly opts out of pretty printing of if this is just a straight up duplicate. But, if its a flag, we should flip default its value to `isatty`.

**Comment by matanlurey** [comment]
Duplicate of https://github.com/flutter/flutter/issues/147903.
